### DISCUSSION TITLE
New Future Action: Do nothing

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#532] [Feature Requests]
> Hello, We recently implemented the "PublishPress Future" plugin which allows us to automatically unpublish WordPress Posts. This plugin allows us to either update the post to 'Draft' or to 'Delete' it. We would like to have the option to 'Do nothing' upon the expirt date being reached as this would allow us to apply Expirt Dates to Page post types. In the case of Page post types it would be good to be able to easily see all expired and about-to-expire Pages in the Page list - but we would not want pages to be moved to Draft or Deleted as this is likely to break the site navigation.

Original request: https://secure.helpscout.net/conversation/2332461593/208901?folderId=6932143

**Comment by andergmartins** [comment]
Related to #384
